### DISCUSSION TITLE
installing cuda

### DISCUSSION THREAD

**Discussion by michael-pruglo** [mrdbourke/pytorch-deep-learning#92] [Q&A] [answered]
I spent an awful lot of time googling and trying different things to enable cuda for pytorch <br>
(I'm on Windows 10) <br>
Tried the official `conda install pytorch torchvision torchaudio cudatoolkit=11.6 -c pytorch -c conda-forge` - I get "torch not compiled with cuda enabled", and in the list of conda packages I have `torch build  py3.8_cpu_0`. <br>
Tried installing everything manually - latest Nvidia driver, Cuda toolkit, cudnn. No Luck. <br>
Tried uninstalling/reinstalling - no luck <br>
Tried uninstalling `cpuonly` - no luck <br>
Tried installing `pytorch-gpu` - not found <br>
<br>
I'm new to conda/package managers, and struggle with installation/configuration a lot. Thought something as popular as PyTorch would be easy. <br>
Appreciate any help. <br>

**Comment by Pouyaexe** [comment]
Are you sure your GPU supports cuda? Many Nvidia GPUs does not. Also did you install the cpu only version before on this environment? I think you have multiple versions installed and that causes the problem. Try creating a new environment and starting fresh using `conda create --name myenv` and thren `conda install pytorch torchvision torchaudio cudatoolkit=11.3 -c pytorch`

**Reply by michael-pruglo** [reply]
I did check my videocard, it does support CUDA. I ran the command with `cudatoolkit`, but it installed cpu-only version for some reason

**Comment by michael-pruglo** [comment, marked as answer]
I switched to Linux and used Mamba - which solved all of my problems, and made CUDA available.

**Reply by mrdbourke** [reply]
Glad to hear you got it fixed @michael-pruglo!
